_sass/messages.scss

```diff
@@ -12,11 +12,19 @@ $error: #BF433A;
   background: transparentize($info, 0.8);
   border-color: $info;
   color: $info;
+
+  p {
```

Comment:
Might also want to remove margin so that it looks the same as the other info boxes.

Reply:
1cbddd6